packages/mobile/locales/es-419/global.json

```diff
@@ -94,7 +94,7 @@
     "0": "Mala Conexión",
     "1": "algunas funciones no disponible"
   },
-  "networkConnectionFailed": "No se pudo conectar a la red, por favor verifica tu conexión",
+  "networkConnectionFailed": "No se pudo conectar a la red, por favor confirmado tu conexión",
```

Comment:
This shouldn't change since the context is different (not talking about phone numbers) `verifica` was the right word